### PR TITLE
Miller Composite Chart

### PR DESCRIPTION
**Pull request by tjwixtrom**
Adds an example of Miller Composite Chart based on [Miller 1972](https://github.com/Unidata/python-gallery/files/1188848/AD0744042.pdf).

### CODE REVIEW

File: examples/miller_composite.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+# With the data now ready, we will create the plot
+
+
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+tlatlons = crs.transform_points(ccrs.PlateCarree(), lon, lat)
```
Comment (dopplershift) [inline examples/miller_composite.py]: How about passing `transform=ccrs.PlateCarree()` to `contour`, etc., and avoid the manual transform?

File: examples/miller_composite.py [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+# Assign variable names to collected data
+dtime = data.variables['Geopotential_height'].dimensions[0]
+dlev = data.variables['Geopotential_height'].dimensions[1]
+lat = data.variables['lat'][:]
+lon = data.variables['lon'][:]
+lev = data.variables[dlev][:] * units.hPa
+times = data.variables[dtime]
+vtimes = num2date(times[:], times.units)
+temps = data.variables['Temperature']
+tmp = (temps[0, :] * units.kelvin)
```
Comment (dopplershift) [inline examples/miller_composite.py]: Parens not needed.

File: examples/miller_composite.py [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+uwnd = data.variables['u_wind'][0, :] * units.meter / units.second
+vwnd = data.variables['v_wind'][0, :] * units.meter / units.second
+hgt = (data.variables['Geopotential_height'][0, :] * units.meter)
```
Comment (dopplershift) [inline examples/miller_composite.py]: Parens not needed.

File: examples/miller_composite.py [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+relh = data.variables['Relative_humidity'][0, :]
+lifted_index = (data.variables['Best_4-layer_lifted_index'][0, 0, :] *
+                units(data.variables['Best_4-layer_lifted_index'].units))
+Td_sfc = (data.variables['Dew_point_temperature'][0, 0, :] *
+          units(data.variables['Dew_point_temperature'].units))
+avor = (data.variables['Absolute_vorticity'][0, :] * units('1/s'))
```
Comment (dopplershift) [inline examples/miller_composite.py]: Parens not needed.

File: examples/miller_composite.py [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+gfsdata2.variables('Geopotential_height',
+                   'Pressure_reduced_to_MSL').add_lonlat()
+
+# Set the lat/lon box for the data you want to pull in.
+gfsdata2.lonlat_box(-135, -60, 15, 65)
+
+# Actually getting the data
+data2 = ncss2.get_data(gfsdata)
+
+hgt_00z = (data2.variables['Geopotential_height'][0, :] * units.meter)
```
Comment (dopplershift) [inline examples/miller_composite.py]: Parens not needed.

**Comment by tjwixtrom** [comment]
Update reflects requested changes.

**Pull request merged**